Tests/SwerverTests/ResponseSpec/HttpResponseSpec.swift

```diff
@@ -5,48 +5,113 @@ import Foundation
 
 class HttpResponseSpec: QuickSpec {
     override func spec() {
-        describe("HttpResponse") {
-            var responseBuilder = ResponseBuilder()
+        describe("A Reponse object") {
+            func buildBuilder() -> HttpResponseBuilder {
+                return HttpResponseBuilder()
+            }
             
-            it("can equate two response object") {
-                let responseOne = responseBuilder
+            func defaultResponseBuilder() -> HttpResponse {
+                return HttpResponseBuilder()
                     .with(statusCode: .ok)
                     .with(contentType: .text)
-                    .with(body: String())
+                    .set(header: .allow, value: "GET")
+                    .with(body: "Hello")
                     .build()
+            }
+            
+            context("when compares with another response object") {
+                it("is true when attributes match") {
+                    var responseBuilder = buildBuilder()
+                    
+                    let responseOne = defaultResponseBuilder()
+                    let responseTwo = responseBuilder
+                        .with(statusCode: .ok)
+                        .with(contentType: .text)
+                        .set(header: .allow, value: "GET")
+                        .with(body: "Hello")
+                        .build()
+                    
+                    expect(responseOne).to(equal(responseTwo))
+                }
                 
-                let responseTwo = responseBuilder
-                    .with(statusCode: .ok)
-                    .with(contentType: .text)
-                    .with(body: String())
-                    .build()
+                it("is false when the status code does not match") {
```

Comment:
This test still has the problems mentioned in my previous comment regarding details in the test.
1. There is no way to know that the status code is different just from looking at the test.
2. There are extraneous details in the test that the test does not care about.

Reply:
fixed